ui/admin/app/templates/scopes/scope/credential-stores/credential-store/edit-worker-filter.hbs

```diff
@@ -0,0 +1,25 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.credential-store.titles.edit-worker-filter.title')}}
+
```

Comment:
Im noticing that this template and the `worker-filter.hbs` template are both missing the `<Breadcrumbs::Item>` component.
For example, for this page (under the page-title) we should have:
`<Breadcrumbs::Item
  @text={{t <stringGoesHere>}}
  @route='scopes.scope.credential-stores.credential-store.edit-worker-filter'
/>`